frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -48,8 +48,9 @@ export const isHelmReleaseNode = (
 };
 
 export const getKialiLink = (consoleLinks: K8sResourceKind[], namespace: string): string => {
```

Comment:
nit: This function returns namespace specific dashboard so may be it's better to rename. Also we may add/enhance this utility later when more touch points are added.

Reply:
Renamed it to `getNamespaceDashboardKialiLink`.